tendrl/gluster_integration/gfapi.py

```diff
@@ -0,0 +1,142 @@
+import ctypes
+from ctypes.util import find_library
+import os
+
+
+GLUSTER_VOL_PROTOCAL = 'tcp'
+GLUSTER_VOL_HOST = 'localhost'
+GLUSTER_VOL_PORT = 24007
+GLUSTER_VOL_PATH = "/"
+
+
+class GlusterLibgfapiException(Exception):
+    message = "Gluster Libgfapi Exception"
+
+    def __init__(self, rc=0, err=()):
+        self.rc = rc
+        self.err = err
+
+    def __str__(self):
+        s = self.message
+        if self.err:
+            e = '\n'.join(self.err)
+            s += '\nerror: ' + e
+        if self.rc:
+            s += '\nreturn code: %s' % self.rc
+        return s
+
+
+class GlfsStatvfsException(GlusterLibgfapiException):
+    message = "Failed to get Gluster volume Size info"
+
+
+class GlfsInitException(GlusterLibgfapiException):
+    message = "glfs init failed"
+
+
+class GlfsFiniException(GlusterLibgfapiException):
+    message = "glfs fini failed"
+
+
+class StatVfsStruct(ctypes.Structure):
+    _fields_ = [
+        ('f_bsize', ctypes.c_ulong),
+        ('f_frsize', ctypes.c_ulong),
+        ('f_blocks', ctypes.c_ulong),
+        ('f_bfree', ctypes.c_ulong),
+        ('f_bavail', ctypes.c_ulong),
+        ('f_files', ctypes.c_ulong),
+        ('f_ffree', ctypes.c_ulong),
+        ('f_favail', ctypes.c_ulong),
+        ('f_fsid', ctypes.c_ulong),
+        ('f_flag', ctypes.c_ulong),
+        ('f_namemax', ctypes.c_ulong),
+        ('__f_spare', ctypes.c_int * 6),
+    ]
+
+
+def glfsInit(volumeId, host, port, protocol):
+    fs = _glfs_new(volumeId)
+
```

Comment:
Could you please add this check: https://github.com/oVirt/vdsm/blob/master/lib/vdsm/gluster/gfapi.py#L65

Reply:
Done